### PR TITLE
docs(common): fix typos in HttpClient docs

### PR DESCRIPTION
**Pull request by nicholasp**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/common/http/src/client.ts [hunk 1]
```diff
@@ -101,9 +101,9 @@ export class HttpClient {
   constructor(private handler: HttpHandler) {}
 
   /**
-   * Sends an `HTTPRequest` and returns a stream of `HTTPEvents`.
+   * Sends an `HttpRequest` and returns a stream of `HttpEvents`.
```
Comment (gkalpak) [inline packages/common/http/src/client.ts:104]: Actually, there is no `HttpEvents` type (only [HttpEvent][1]). Let's change this to:

```suggestion
   * Sends an `HttpRequest` and returns a stream of `HttpEvent`s.
```

(Same for other occurrences of `HttpEvents` below.)

[1]: https://angular.io/api/common/http/HttpEvent

File: packages/common/http/src/client.ts [hunk 2]
```diff
@@ -173,7 +173,7 @@ export class HttpClient {
    * @param url     The endpoint URL.
    * @param options The HTTP options to send with the request.
    *
-   * @return An `Observable` of the response, with the response body as an array of `HTTPEvents` for
+   * @return An `Observable` of the response, with the response body as an array of `HttpEvents` for
    *     the
    * request.
```
Comment (gkalpak) [inline packages/common/http/src/client.ts:178]: While you are at it, feel free to get rid of the unnecessary newline here:

```suggestion
   *     the request.
```

**Review by gkalpak — CHANGES_REQUESTED** [review]
Thank you, @nicholasp :100:
I made a couple of minor suggestions (otherwise lgtm :+1:).

**Comment by nicholasp** [comment]
Hi, @gkalpak ! Thanks I did fixup

File: packages/common/http/src/client.ts [hunk 3]
```diff
@@ -101,9 +101,9 @@ export class HttpClient {
   constructor(private handler: HttpHandler) {}
 
   /**
-   * Sends an `HttpRequest` and returns a stream of `HttpEvents`.
+   * Sends an `HttpRequest` and returns a stream of `HttpEvent`.
```
Comment (gkalpak) [inline packages/common/http/src/client.ts:104]:
```suggestion
   * Sends an `HttpRequest` and returns a stream of `HttpEvent`s.
```

File: packages/common/http/src/client.ts [hunk 4]
```diff
@@ -173,9 +173,8 @@ export class HttpClient {
    * @param url     The endpoint URL.
    * @param options The HTTP options to send with the request.
    *
-   * @return An `Observable` of the response, with the response body as an array of `HttpEvents` for
-   *     the
-   * request.
+   * @return An `Observable` of the response, with the response body as an array of `HttpEvent` for
+   * the request.
```
Comment (gkalpak) [inline packages/common/http/src/client.ts:177]:
```suggestion
   *     the request.
```

**Review by gkalpak — CHANGES_REQUESTED** [review]
Thx for making the changes, @nicholasp! Seems like you missed an "s" at the end :smiley:

Note, we still want to have the "s" (for grammatical conrectness), but we want it to be outside the backticks (i.e. outside the generated `<code>` blocks).

**Review by gkalpak — CHANGES_REQUESTED** [review]
Nice work, @nicholasp ✨
I've left a few more comments with typos to be fixed 😇

**Comment by nicholasp** [comment]
@gkalpak  :D 👍

File: packages/common/http/src/client.ts [hunk 5]
```diff
@@ -2406,7 +2405,7 @@ export class HttpClient {
    * @param body The resources to add/update.
    * @param options HTTP options
    *
-   * @return An `Observable` of all HttpEvents for the request, with a response body
+   * @return An `Observable` of all `HttpEvent` for the request, with a response body
```
Comment (gkalpak) [inline packages/common/http/src/client.ts:2408]:
```suggestion
   * @return An `Observable` of all `HttpEvent`s for the request, with a response body
```

File: packages/common/http/src/client.ts [hunk 6]
```diff
@@ -1203,8 +1202,8 @@ export class HttpClient {
    * @param url     The endpoint URL.
    * @param options The HTTP options to send with the request.
    *
-   * @return An `Observable` of all HttpEvents for the request, with the response body of type
-   *     string.
+   * @return An `Observable` of all `HttpEvent` for the request, with the response body of type
```
Comment (gkalpak) [inline packages/common/http/src/client.ts:1205]:
```suggestion
   * @return An `Observable` of all `HttpEvent`s for the request, with the response body of type
```

**Review by gkalpak — CHANGES_REQUESTED** [review]
Last two comments :grin:
(BTW, thx for adding the changes in fixup commits. It makes it so much easier to review :100:)

**Comment by nicholasp** [comment]
:) thanks also for guiding me @gkalpak

**Review by gkalpak — APPROVED** [review]
The changes look great 🚀 :100: ✨
It seems like you need to rebase on latest master one last time :wink:

Reviewed-for: global-docs-approvers

**Pull request closed without merging**